src/index.js

```diff
@@ -1,186 +1,29 @@
-const addTasksBtn = document.getElementById('add-task-btn');
-const deskTaskInput = document.getElementById('description-task');
-const todoWrapper = document.querySelector('.todos-wrapper');
-
-let tasks;
-/* eslint-disable */
-!localStorage.tasks ? tasks = [] : tasks = JSON.parse(localStorage.getItem('tasks')); 
-/* eslint-anable */
-
-let todoItemElems = [];
-
-function Task(description) {
```

Comment:
- [ ] In this project you are required ES6 module and syntaxes. It is recommended to use the arrow function whenever necessary. Kindly refactor this and other functions to arrow functions. Kindly make the changes to all functions that can be written by the arrow function.  :pray:

Reply:
Dear @henatan99! 🥇 

Many thanks for the review. I appreciate your comments 👍 

It seems you are looking not for the correct branch. The PR is related to "bacis-page-structure"

Many thanks and regards,

Elmar

Reply:
HI @elmar8287, my reviewer dashboard link brought me here. I don't think it is my mistake. Maybe you provided the wrong link when you submit your project. Sorry for the mishap and if this is the wrong link, reach out to student success please.

Reply:
Dear @henatan99,

I submit one more time and got approved. Maybe there was a wrong link. Thanks for review